### ISSUE TITLE
Version Numbering Is Off

### ISSUE THREAD

**Issue by CG-Tespy** [RealityStop/Bolt.Addons.Community#70]
At the time of this writing, the latest is version 3.2. But when updating through the Package Manager, it said that the latest version was 4.0.5. 

![image](https://github.com/RealityStop/Bolt.Addons.Community/assets/22418199/c7d510fd-0815-4fee-b9eb-f27a9437f244)

**Comment by S2NX7** [comment]
Yeah that's how it was when i first started working on it, i am not sure if it will break things if i change it to 3.2

**Comment by CG-Tespy** [comment]
Fair. Then for clarity's sake, upon every new release of this asset package, can you include a note about the version number Unity will tell you it is? It'd help avoid confusion

**Comment by S2NX7** [comment]
> Fair. Then for clarity's sake, upon every new release of this asset package, can you include a note about the version number Unity will tell you it is? It'd help avoid confusion

Ok i can do that.
